02-library/cs506/sim.py

```
@@ -1,13 +1,56 @@
+import math
 def euclidean_dist(x, y):
-    raise NotImplementedError()
+    if len(x) != len(y):
+        return "Vectors have different dimention"
```

you should raise an error instead of returning a string